packages/DefaultAutowire/src/DependencyInjection/DefinitionAnalyzer.php

```diff
@@ -138,14 +137,18 @@ private function haveMissingArgumentsTypehints(
                     continue;
```

Comment:
If it makes test pass, why not. I'm not sure what the use case was. If it isn't covered, it probably wasn't revelant.